geps/gep-1619/index.md

```diff
@@ -314,36 +346,74 @@ these two configurations.
+`BackendLoadBalancePolicy` could also be later expanded to contain [session affinity](#the-relationship-of-session-persistence-and-session-affinity)
+configuration. This would provide a convenient grouping of the two related APIs within the same policy object.
+Additionally, other future enhancements to the API may include the addition of timeouts, connection draining, and
+logging within `BackendLoadBalancePolicy`.
 
 ```go
-// SessionPersistencePolicy provides a way to define session persistence rules
-// for a service or route.
-//
-// Support: Core
-type SessionPersistencePolicy struct {
+// BackendLoadBalancePolicy provides a way to define load balancing rules
+// for a backend.
+type BackendLoadBalancePolicy struct {
     metav1.TypeMeta   `json:",inline"`
     metav1.ObjectMeta `json:"metadata,omitempty"`
 
-    // Spec defines the desired state of SessionPersistencePolicy.
-    Spec SessionPersistencePolicySpec `json:"spec"`
+    // Spec defines the desired state of BackendLoadBalancePolicy.
+    Spec BackendLoadBalancePolicySpec `json:"spec"`
 
-    // Status defines the current state of SessionPersistencePolicy.
-    Status SessionPersistencePolicyStatus `json:"status,omitempty"`
+    // Status defines the current state of BackendLoadBalancePolicy.
+    Status PolicyStatus `json:"status,omitempty"`
 }
 
-// SessionPersistencePolicySpec defines the desired state of
-// SessionPersistencePolicy.
+// BackendLoadBalancePolicySpec defines the desired state of
+// BackendLoadBalancePolicy.
 // Note: there is no Override or Default policy configuration.
-type SessionPersistencePolicySpec struct {
+type BackendLoadBalancePolicySpec struct {
     // TargetRef identifies an API object to apply policy to.
-    // The TargetRef may be a Service, HTTPRoute, GRPCRoute,
-    // or a HTTPRouteRule or GRPCRouteRule section.
-    // At least one of these targets must be supported for
-    // core-level compliance.
-    //
+    // Currently, Backends (i.e. Service, ServiceImport, or any
+    // implementation-specific backendRef) are the only valid API
+    // target references.
     TargetRef gatewayv1a2.PolicyTargetReference `json:"targetRef"`
 
+    // SessionPersistence defines and configures session persistence
+    // for the backend.
+    //
+    // Support: Extended
+    //
+    // +optional
+    SessionPersistence *BackendSessionPersistence `json:"sessionPersistence"`
+}
+
+// BackendSessionPersistence defines the desired state of
+// SessionPersistence for backends.
+type BackendSessionPersistence struct {
+    // SessionName defines the name of the persistent session token
+    // which may be reflected in the cookie or the header.
+    //
+    // Support: Implementation-specific
+    //
+    // +optional
+    // +kubebuilder:validation:MaxLength=4096
+    SessionName *string `json:"sessionName,omitempty"`
+
     // AbsoluteTimeoutSeconds defines the absolute timeout of the
```

Comment:
Is there a way to provide a default value to `AbsoluteTimeoutSeconds`?  I assume some implementations MAY not support this and would like to turn it off by default, so it would be nice if we can have a default value to say "no absolute timeout of the persistent session".